packages/config/config/devices/0x0330/zv2833pac.json

```diff
@@ -0,0 +1,211 @@
+{
+	"manufacturer": "Sunricher",
+	"manufacturerId": "0x0330",
+	"label": "ZV2833PAC",
+	"description": "Push-Button Coupler",
+	"devices": [
+		{
+			"productType": "0x0400",
+			"productId": "0xd105"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"maxNodes": 5,
+			"isLifeline": true
+		},
+		"2": {
+			"label": "Launch 1",
```

Comment:
Do we know what those additional associations do? Maybe there's a better name? Or are these just 4 control groups?

Reply:
From the manual posted, these are four control groups that do different things based on the based on the switch type - param 2 and function param 9.  This is a complicated device and the owner is going to have to read the manual several times (IMO)